app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx

```diff
@@ -26,21 +35,51 @@ export const ListingDetailsAdditionalInformation = ({
             <TextTruncate text={listing.Listing_Other_Notes} />
           </div>
         )}
-        <div className="info-card bg-gray-100 border-0">
-          <h3 className="text-serif-lg">{t("listings.requiredDocuments")}</h3>
-          <div className="text-sm">
-            <TextTruncate text={listing.Required_Documents} />
+        {listing.Required_Documents && (
+          <div className="info-card bg-gray-100 border-0">
+            <h3 className="text-serif-lg">{t("listings.requiredDocuments")}</h3>
+            <div className="text-sm">
+              <TextTruncate text={listing.Required_Documents} />
+            </div>
+          </div>
+        )}
```

Comment:
Any idea what's going on for Required Documents section for [this listing](https://dahlia-webapp-pr-1701.herokuapp.com/listings/a0W0P00000GlKfBUAV?react=true)?
![image](https://user-images.githubusercontent.com/98352600/186546167-764b50eb-99c1-4bb7-98e8-3dfeaae004b3.png)

Reply:
Looks like an issue with how the `Markdown` package is modifying the string (`Proof of residence\r\nBank statements\r\nConcert tickets`) - thoughts on how Markdown is handled elsewhere? I thought I remember this was maybe something we wanted to remove?

Reply:
According to the readme, looks like the [markdown-to-jsx](https://github.com/probablyup/markdown-to-jsx) lib doesn't preserve line breaks.

We may need to copy over something similar to [what angular was doing](https://github.com/SFDigitalServices/sf-dahlia-web/blob/83ce5a75469b737a8c415a5d92d862b68b6cfe3e/app/assets/javascripts/customFilters.js.coffee#L27-L27). Feel free to file a separate ticket if u feel this is outside scope of this story. Just happened to catch it while testing here.

Reply:
Thanks for fixing this!